pkgs/development/libraries/vte/default.nix

```diff
@@ -45,6 +46,8 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  mesonFlags = lib.optionals (!systemdSupport) [ "-D_systemd=false" ];
```

Comment:
Could you please move this under the inputs and put the list item on a separate line like with the inputs?

Reply:
Sorry about that!  Still learning the nixpkgs coding style.  Latest push should fix this.  Thanks for the prompt review!

Reply:
Sorry, by inputs I meant `propagatedBuildInputs` too.

It is actually not an official rule for Nixpkgs – it was observed as a common pattern in Nixpkgs, and some parts of the ecosystem (e.g. GNOME packaging) have stricter coding style that [incorporated it as a rule](https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md). It is a bit unfortunate that we have differing coding styles throughout Nixpkgs but package sets like GNOME contain so many packages that such extra rules are necessary to make maintenance feasible with the limited amount of maintainers. At least for GNOME, you can use [nixpkgs-hammering](https://github.com/jtojnar/nixpkgs-hammering) to check that.